### PR TITLE
*: use std/slices to replace exp/slices

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: ref #45933

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/duplicate/worker.go [hunk 1]
```diff
@@ -17,13 +17,13 @@ package duplicate
 import (
```
Comment (hawkingrei) [inline br/pkg/lightning/duplicate/worker.go:17]: Please run ```make bazel_prepare```.

**Review by hawkingrei — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**